test/integration/datadog_kubernetes/serverspec/kubernetes_spec.rb

```diff
@@ -0,0 +1,33 @@
+# Encoding: utf-8
```

Comment:
instead of an integration test here, could you write a spec test? (you can take https://github.com/DataDog/chef-datadog/blob/v2.8.1/spec/integrations/docker_daemon_spec.rb as an example)

Spec tests are way lighter to run and can cover the integration recipes very well